### PR TITLE
Set flags for existing return version

### PR DESCRIPTION
**Pull request by jonathangoulding**
https://eaflood.atlassian.net/browse/WATER-4731

When the copying existing requirements and using previous requirements for returns, the quarterly returns and multiple uploads flags should inherit from the existing return versions.

This takes precedence over the previous defaults set  earlier in the journey.

### CODE REVIEW

File: app/services/return-versions/setup/existing/generate-from-existing-requirements.service.js [hunk 1]
```diff
@@ -20,13 +20,17 @@ const FetchExistingRequirementsService = require('./fetch-existing-requirements.
  *
  * @param {string} returnVersionId - The UUID of the selected return version to copy requirements from
  *
- * @returns {Promise<object[]>} an array of return requirements generated from the existing return version and ready to
- * be persisted to the setup session
+ * @returns {Promise<object>}  - return an array of return requirements generated from the existing return version and ready to
+ * be persisted to the setup session, if the return version has multiple uploads and if the return version is for quarterly returns
```
Comment (Cruikshanks) [inline app/services/return-versions/setup/existing/generate-from-existing-requirements.service.js:24]: Thank you for correcting the return type. But might I suggest ...

```suggestion
 * @returns {Promise<object>} the return version's return requirements transformed into an array of session return
 * requirements ready to be persisted to the setup session, plus whether the return version allows multiple uploads and
 * quarterly returns
```

**Review by Cruikshanks — CHANGES_REQUESTED** [review]
Just checking whether a test has been inadvertently duplicated, plus a NIT.

File: test/services/return-versions/setup/existing/generate-from-existing-requirements.service.test.js [hunk 2]
```diff
@@ -24,6 +24,96 @@ describe('Return Versions Setup - Generate From Existing Requirements service',
...
+              '56-returns-exception',
+              'gravity-fill',
+              'transfer-re-abstraction-scheme',
+              'two-part-tariff'
+            ]
+          }
+        ])
+      })
+    })
+
```
[the remark this answers is not in the document]
Comment (jonathangoulding) [reply test/services/return-versions/setup/existing/generate-from-existing-requirements.service.test.js]: Correct. Not sure how that happened.

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**